date_module.js

```diff
@@ -0,0 +1,88 @@
+//setdate
+// set a variable for the date using built in js date function
+// setDate is a method that allows the value of the date variable to be changed by user input
+// user enters a new date which is saved to variable
+
+var myObj = ( function iife() {
+  var myDate = new Date();
```

Comment:
Rather than instantiate the internally stored date immediately upon executing your iife, it would be better to initialize this to null and only set it when it's actually needed - i.e. when one of the module interface methods is first called.